### DISCUSSION TITLE
Import document does nothing

### DISCUSSION THREAD

**Discussion by TNBRE** [open-webui/open-webui#1296] [General]
# Bug Report

## Description

**Bug Summary:**
[Open webui don't seems to load documents for RAG]

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

**Expected Behavior:**
[Describe what you expected to happen.]

**Actual Behavior:**
[Describe what actually happened.]

## Environment

- **Operating System:** Ubuntu 20.04
- **Browser (if applicable):** [Edge]

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

Docker

## Additional Information

Ollama is running with Llama 2 and i,m able to check it throug the Ollama ip address. I can chat with my model in Open Webui but i cannot upload any document and ask questions.

When i try to upload a document in the documents section nothing happends

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by pwab** [comment]
As far as I know the document is embedded in the background - and this process needs some time depending on your hardware. I tried it right now on my system with an 1.5mb .txt-file and it needed about 5 minutes while only using the cpu (because my laptop has no dedicated gpu).

But it would be a good ui improvement to visualize the embedding process in the document uploading section.
